services/kafka.go

```diff
@@ -95,12 +95,12 @@ func (k *KafkaService) StartBackgroundSend(sendWaitGroup *sync.WaitGroup, shutdo
 				Payload:   base64.StdEncoding.EncodeToString(payload.Data),
 				Metadata:  "{}",
 			}
-			// log.Printf("Encoded data: %s\n", messageValue.Payload)
+			// zap.S().Infof("Encoded data: %s", messageValue.Payload)
```

Comment:
Spurious comment?

Reply:
now that it is level managed, we can uncomment